tests/cupyx_tests/scipy_tests/stats_tests/test_morestats.py

```diff
@@ -141,3 +141,23 @@ def test_array_empty_neg_lmb(self, xp, scp, dtype):
             return numpy.array(result)
         else:
             return result
+
+
+class TestKstat:
+    def test_moments_normal_distribution(self):
```

Comment:
We have a helper decorator `cupy.testing.for_all_dtypes` to make a parametrized test across dtypes. Would you use it to check with other dtypes?